runelite-client/src/main/resources/net/runelite/client/plugins/gpu/vert.glsl

```diff
@@ -121,5 +49,6 @@ void main()
 
   vPosition = vertex;
   vColor = vec4(rgb, 1.f - a);
+  vHsl = vec2((hsl >> 8)/255.f, (hsl & 0xff)/255.f);
```

Comment:
Does this work the same way if you just use an ivec2? or an int?

Reply:
You can't have ivec2 as a vertex attribute because then the attribute needs to be flat(the entire triangle shares one value). If it shares one value then there's the entire triangle is going to share the same rgb value too.